main_test.go

```diff
@@ -0,0 +1,26 @@
+package main
```

Comment:
You shouldn't test private functions anyway, instead you should test their side effects. If the functionality is too complex that deserves a test, it may need to transform it in another component.